packages/mobile/test/schemas.ts

```diff
@@ -581,6 +581,14 @@ export const v9Schema = {
       CodeInputStatus.Disabled,
     ],
   },
+  networkInfo: {
+    ...v8Schema.networkInfo,
+    userLocationData: {
+      countryCodeAlpha2: 'US',
+      region: null,
+      ipAddress: null,
+    },
+  },
```

Comment:
This should be moved to a new schema.
Actually for good practice, we should match it the redux store version number, so `v13Schema`:
https://github.com/celo-org/wallet/blob/0ac68cf5a5fdff7762543a8687f2e90778afbed4/packages/mobile/test/schemas.ts#L369-L370

This makes it easier to test migrations, and avoid confusions with the version numbers.